interop.md

```diff
@@ -391,3 +392,37 @@ public class JavaClass {
     }
 }
 ```
+# Lint Checks for Interoperability
+
+## Requirements
+* **Android Studio version required:** 3.2 Canary 10 or later
+* **Lint version required:** 26.2.0-alpha10 or later
```

Comment:
People don't really think of the lint version using the internal sdk library version numbers; I'd instead reference this as Gradle version required, which would be 3.2.0-alpha10 or later

Reply:
I wouldn't even mention anything but "3.2"

Reply:
For people who have an earlier version of the Canary, they may try it and it doesn't work. I don't want people to getting confused, so I think it's better to be explicit. Once we go Beta/Stable, we can update.